python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_ssh.schema.yml

```diff
@@ -10,6 +10,22 @@ keys:
   management_ssh:
     type: dict
     keys:
+      authentication:
+        type: dict
+        keys:
+          empty_passwords:
+            type: str
+            valid_values: ["auto", "deny", "permit"]
+            description: |
+              setting to allow or deny empty passwords for ssh authentication
+            default: "auto"
+          protocol:
```

Comment:
Usually we use the plural form for lists. So I suggest to change this to `protocols` all over.